style.css

```diff
@@ -27,8 +28,29 @@ body {
   margin: 0;
   padding: 0;
   list-style-type: none;
+ }
+
+.topic li 	{
+	margin: 10px auto;
+	width: 900px;
+}
+
+.topic-title {
+	color: dark blue;
 }
+/* changing the list to boxes - it's not pretty but I wanted to practise Git*/
+li {
+	width: 300px;
+	display: inline-block;
```

Comment:
You'll have noticed that smaller `li` gets aligned to the bottom. There is `vertical-align` property that lets you change this however.

```css
vertical-align: top;
```